.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
-  - sudo pip3 install -r requirements.txt
+  - sudo pip3 install -U -r requirements.txt
```

Comment:
@Uran198 Little confused here. Ideally `colorama~=0.3.6` is installed from [PyPrint](https://github.com/coala-analyzer/PyPrint/blob/master/requirements.txt#L2). Should this problem get solved anyway without further upgrading?